src/require-strict-assert.spec.ts

```diff
@@ -0,0 +1,107 @@
+// require-strict-assert.spec.ts
+
+/*
+ * Copyright (c) 2021-2022 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+import { RuleTester } from 'eslint';
+
+import rule from './require-strict-assert';
+
+describe('require-strict-assert', () => {
```

Comment:
i'd like to see tests for all the possible "strict" methods, including e.g. `assert.notDeepStrictEqual` and `assert.notStrictEqual`